File: ios/Plugin/CloudinaryHelper.swift
import Foundation
import Cloudinary
import Capacitor

public class CloudinaryHelper {

    public static func createUploadResourceResult(_ resultData: CLDUploadResult) -> JSObject {
        var result = JSObject()
        result["createdAt"] = resultData.createdAt
        result["format"] = resultData.format
        result["width"] = resultData.width
        result["height"] = resultData.height
        result["originalFilename"] = resultData.originalFilename
        result["resourceType"] = resultData.resourceType
        result["publicId"] = resultData.publicId
        result["url"] = resultData.url
        return result
    }
}

File: android/src/main/java/io/capawesome/capacitorjs/plugins/cloudinary/CloudinaryHelper.java
package io.capawesome.capacitorjs.plugins.cloudinary;

import com.getcapacitor.JSObject;
import java.util.Map;

public class CloudinaryHelper {

    public static JSObject createUploadResourceResult(Map resultData) {
        JSObject result = new JSObject();
        result.put("assetId", resultData.get("asset_id"));
        result.put("bytes", resultData.get("bytes"));
        result.put("createdAt", resultData.get("created_at"));
        result.put("format", resultData.get("format"));
        result.put("originalFilename", resultData.get("original_filename"));
        result.put("resourceType", resultData.get("resource_type"));
        result.put("publicId", resultData.get("public_id"));
        result.put("url", resultData.get("url"));
        return result;
    }
}
